.github/workflows/release-k8s.yaml

```
@@ -0,0 +1,16 @@
+name: Release k8s
+
+on:
+  push:
+    tags: ["v*"]
+
+jobs:
+  release:
+    name: Release Contrib
+    uses: ./.github/workflows/base-release.yaml
+    with:
+      distribution: otelcol-k8s
+      goos: '[ "linux" ]'
+      goarch: '[ "amd64", "arm64", "ppc64le", "s390x" ]'
+    secrets: inherit
+    permissions: write-all
```

```suggestion
    permissions: write-all

```
Nit. Having a trailing newline to get rid of the "no newline" symbol would be nice.